static/js/pages/PodcastFrontpage.js

```diff
@@ -43,14 +45,18 @@ export default function PodcastFrontpage() {
           </>
         ) : null}
       </div>
-      <h1>PODCASTS</h1>
-      {loaded
-        ? Object.values(podcasts).map((podcast: any, idx) => (
-          <div key={idx}>
-            <h3>{podcast.title}</h3>
-          </div>
-        ))
-        : null}
+      <div className="all-podcasts">
+        <h1>Podcasts Series</h1>
+        {loaded ? (
+          <Grid>
+            {Object.values(podcasts).map((podcast: any, idx) => (
+              <Cell width={4} key={idx}>
```

Comment:
Can we use `podcast.id` instead of `idx`? It might be more resilient if we do end up using infinite scroll at some point in the future

Reply:
sure